examples/v3.0/api-with-examples.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: api-with-examples
+parent: Example API Descriptions
+---
+
+{% comment %}
+{% capture description %}
+Insert description here, then remove comment tag above and endcomment tag below
```

Comment:
Do we need to fill in the descriptions? I see a number of them that contain this boilerplate text.

Reply:
We don't have to, but it may be nice to add a sentence or two on what the example is supposed to show/teach.

I didn't have the creativity today to fill in these blanks, so I added the placeholders.

Feel free to add them to this PR, or to subsequent PRs once this is merged.

Reply:
@lornajane created #106 for this